src/main/resources/META-INF/definitions.cnd

```diff
@@ -3,6 +3,14 @@
 
 [jseont:sitemap] > jnt:content
 
+[jseont:sitemapResource] > jnt:content
+
 [jseomix:sitemap] mixin
  extends = jnt:virtualsite
  + sitemap (jseont:sitemap) = jseont:sitemap mandatory autocreated
+
+[jseomix:sitemapResource] mixin
+ extends = jnt:page, jmix:mainResource
+ - jseomix:noIndex (boolean)
```

Comment:
Maybe we can simplify the naming of properties a bit and remove jseomix prefix, just a suggestion I think it may make it a bit more readable and less confusing because we won't confuse it with an actual mixin.

Reply:
Removed prefixes on attributes